src/main/java/ladder/domain/LadderLine.java

```diff
@@ -0,0 +1,48 @@
+package ladder.domain;
+
+import ladder.exception.InvalidRopeException;
+
+import java.util.Iterator;
+import java.util.List;
+import java.util.Objects;
+import java.util.stream.IntStream;
+
+public class LadderLine implements Iterable<Rope> {
+    private final List<Rope> ropes;
+
+    public LadderLine(List<Rope> ropes) {
+        validateContinuousRope(ropes);
+        this.ropes = ropes;
+    }
+
+    private void validateContinuousRope(List<Rope> ropes) {
+        boolean valid = IntStream.range(0, ropes.size() - 1)
+                .noneMatch(iRopeIndex ->
+                        ropes.get(iRopeIndex).isPresent() && ropes.get(iRopeIndex + 1).isPresent()
+                );
+        if (!valid)
+            throw new InvalidRopeException("이어지는 로프가 있습니다.");
```

Comment:
```suggestion
        if (!valid) {
                throw new InvalidRopeException("이어지는 로프가 있습니다.");
        }   
```

자바에서는 하나의 라인에도 괄호를 써주는 것이 관례입니다 :)

Reply:
컨벤션을 지키려고 하는데 습관이 굳어져서 자꾸 빼먹는 실수가 발생 하네요👏